client/js/stores/month-store.js

```diff
@@ -2,8 +2,33 @@ var Dispatcher = require('./dispatcher/dispatcher');
 var AppConstants = require('./constants/constants');
 var EventEmitter = require('event').EventEmitter;
 
-var MonthStore = {
+var _MonthEvents = [];
 
-};
+for(var i=0; i<28; i++) {
+  _MonthEvents.push({
+    eventName:'wedding',
+    eventLocation: 'DC',
+    requiredStaff: 100,
+    time: '10:00AM'
+  });
+}
+
+var MonthStore = assign(EventEmitter.prototype, {
```

Comment:
this will copy all of the second objects properties onto the EventEmitter which is not what we want. you should set the target to be an empty object like: assign({}, EventEmitter.prototype, { .... }); Also, where does assign get defined? Also, I noticed you changed the name of the constants file, you will need to change the require statement on line 2.